android/app/build.gradle

```diff
@@ -24,7 +24,7 @@ if (flutterVersionName == null) {
 
 android {
     compileSdk 34
-    ndkVersion flutter.ndkVersion
+    ndkVersion "26.1.10909125"
```

Comment:
This should be safe to remove, see

https://github.com/flutter/flutter/issues/139427#issuecomment-1989024985